engine/commands/cortex_upd_cmd.h

```diff
@@ -1,13 +1,124 @@
 #pragma once
-#include <string>
 #include <optional>
+#include <string>
+
+#include "httplib.h"
+#include "nlohmann/json.hpp"
+#include "utils/file_manager_utils.h"
+#include "utils/logging_utils.h"
 
 namespace commands {
+#ifndef CORTEX_VARIANT
+#define CORTEX_VARIANT file_manager_utils::kProdVariant
+#endif
+constexpr const auto kNightlyHost = "https://delta.jan.ai";
+constexpr const auto kNightlyFileName = "cortex-nightly.tar.gz";
```

Comment:
@hiento09 Isn't this filename supposed to be `cortexcpp-nightly.tar.gz`? I'm ok either way, just need to be sure

Reply:
We discussed about nightly updater here @dan-homebrew  https://github.com/janhq/cortex.cpp/discussions/1116#discussioncomment-10566504